langchain/document_loaders/recursive_url_loader.py

```diff
@@ -83,19 +90,20 @@ def get_child_links_recursive(
             # Check all unvisited links
             if link not in visited:
                 visited.add(link)
+                loaded_link = WebBaseLoader(link).load()
+                if isinstance(loaded_link, list):
```

Comment:
can this use lazy_load as well? or no?

Reply:
`WebBaseLoader` is currently only loading a single page here